porfile.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Portfolio</title>
+    <link rel="stylesheet" href="style.css">
+</head>
+
+<body>
+    <header>
+        <nav class="navbar">
+            <a href="index.html" class="brand">My Logo</a>
+            <button id="hamburger"><img width="16" src="Assets/icons8-hamburger-32 (1).png" alt="Menu"></button>
+        </nav>
+    </header>
+    <section class="headline">
+        <div class="content">
+            <h1 class="title">
+                I'm Samuel ogoigbe
+                <br>
+                Glad to see you!
+            </h1>
+            <p class="description">
+                I'm a software developer! I can help you build a product , feature or website Look through some of my
+                work
+                and
+                experience! If you like what you see and have a project you need coded, don't hestiate to contact me.
+            </p>
+            <p class="connect-link">
+                LET'S CONNECT
```

Comment:
- [ ] As observed in the Figma design template, the headline section has a background image. Please add the proper background image (decorations are always background images, not `img` tags). Usually, if an image is not part of the content (aesthetic reasons) use CSS background-image. [Check this link](https://stackoverflow.com/questions/492809/when-to-use-img-vs-css-background-image) for more info.
- [ ] The left bottom border-radius in your design is not similar to that of the Figma template. Kindly fix this.

Your Design | Figma Design
---- | ----
![image](https://user-images.githubusercontent.com/84156714/220061704-f1b56199-f842-4dc5-805d-d79fab0e11f7.png) | ![image](https://user-images.githubusercontent.com/84156714/220062560-be00df27-f4a8-41e3-8204-1dadc1307b52.png)